stubs/django-filter/django_filters/rest_framework/backends.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+from django.utils.deprecation import RenameMethodsBase
+
+from .. import compat as compat, utils as utils
+from . import filters as filters, filterset as filterset
```

Comment:
Should I convert relative imports to absolute? Or I could do that in follow-up as well.

Reply:
We're pretty inconsistent about this across typeshed at the moment, so either is fine. Ideally the files in the `stubs/django-filter/` directory would pick a style and stick to it, but even there, tbh I don't really care that much about it

Reply:
I *personally* just use whatever I copy-pasted from the source, or my editor auto-imported. Sometimes I prefer using relative imports on purpose when my editor (ie: VSCode with Pylance) can't pickup a newly added module to typeshed stubs.

As @AlexWaygood said, we're pretty inconsistent about it.